src/pages/user/login/components/Login/LoginItem.tsx

```diff
@@ -19,23 +19,24 @@ export interface LoginItemType {
   Captcha: React.FC<WrappedLoginItemProps>;
 }
 
-export interface LoginItemProps extends GetFieldDecoratorOptions {
+export interface LoginItemProps {
   name?: string;
+  rules?: any[];
   style?: React.CSSProperties;
-  onGetCaptcha?: (event?: MouseEvent) => void | Promise<boolean> | false;
+  onGetCaptcha?: (event?: MouseEvent) => void | Promise<any> | false;
   placeholder?: string;
   buttonText?: React.ReactNode;
-  onPressEnter?: (e: React.KeyboardEvent<HTMLInputElement>) => void;
+  onPressEnter?: (e: any) => void;
   countDown?: number;
   getCaptchaButtonText?: string;
   getCaptchaSecondText?: string;
   updateActive?: LoginContextProps['updateActive'];
   type?: string;
   defaultValue?: string;
   form?: FormComponentProps['form'];
-  customProps?: { [key: string]: unknown };
-  onChange?: (e: React.ChangeEvent<HTMLInputElement>) => void;
-  tabUtil?: LoginContextProps['tabUtil'];
+  customProps?: { [key: string]: any };
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type.](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=4583897)